includes/class-notices.php

```diff
@@ -167,7 +167,7 @@ public function render_admin_notices() {
 
 			$css_id = ( false === strpos( $notice['id'], 'give' ) ? "give-{$notice['id']}" : $notice['id'] );
 
-			$css_class = "give-notice notice is-dismissible {$notice['type']} notice-{$notice['type']}";
+			$css_class = "give-notice notice ". ( 'error' === (string) $notice['type'] ? 'non': 'is' ) ."-dismissible {$notice['type']} notice-{$notice['type']}";
```

Comment:
Hmm, I don't think we want all `error` types as non-dismissible. We should discuss with @ravinderk

Reply:
@DevinWalker @raftaar1191 I think we can create `is_dismissible` as notice params where by default this is set to `true` and developer can use this.

@DevinWalker Let me know what do you think?